test/api/assignments_test.rb

```diff
@@ -72,7 +72,7 @@ def test_provides_a_useful_error_message_when_key_is_wrong
   def test_api_accepts_submission_on_completed_exercise
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
-        post '/user/assignments', { key: alice.key, code: 'THE CODE', path: 'go/one/code.go', 
+        post '/user/assignments', { key: alice.key, code: 'THE CODE', path: 'go/one/code.go',
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.